subprojects/core-api/src/main/java/org/gradle/api/NamedDomainObjectContainer.java

```diff
@@ -28,6 +28,8 @@
  * <p>Note that a container is an implementation of {@link java.util.SortedSet}, which means that the container is guaranteed
  * to only contain elements with unique names within this container. Furthermore, items are ordered by their name.</p>
  *
+ * @apiNote Implementations can be created using {@link Project#container} or it's variants.
```

Comment:
I think the version of javadoc we use doesn't support this tag.  Could you switch this to `@see` and link directly to `Project#container(Class)` and `Project#container(Class,NamedDomainObjectFactory)`?

Reply:
Sure, I wasn't sure if you actually wanted this to show up in the generated docs.
I'll change it.

Do you not want me to say "Implementations can be created using" somehow or just drop that?

Reply:
Sort of, I was thinking you could do something like:
> @see Project#container(Class) Creating a container
> @see Project#container(Class, NamedDomainObjectFactory) Creating a container with a custom factory

And that would render as clickable links.

Reply:
Will do.